pi-hole.json

```diff
@@ -0,0 +1,83 @@
+                        "host_default": 80,
+                        "label": "Web-Port",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/etc/pihole": {
+                        "description": "Choose a Share for Pi-Hole configuration. Eg: create a Share called pihole-config for this purpose alone.",
+                        "label": "Config Storage"
+                    }
+                },
+                "environment": {
+                    "ServerIP": {
+                        "description": "Enter IP-adress of rockstor server. If not specified it will default to internal docker IP and it will not work.",
+                        "label": "Rockstor IP",
+                        "index": 1
+                    },
+                    "VIRTUAL_HOST": {
+                        "description": "Enter IP of rockstor server.",
+                        "label": "Virtual Host",
+                        "index": 2
+                    },
+                    "DNS1": {
+                        "description": "Enter Primary DNS server. Eg: 8.8.8.8",
+                        "label": "Primary DNS Server",
+                        "index": 3,
+                        "default": "8.8.8.8"
+                    },
+                    "DNS2": {
+                        "description": "Enter Secondary DNS server. Eg: 8.8.4.4",
+                        "label": "Secondary DNS Server",
+                        "index": 4,
+                        "default": "8.8.4.4"
+                    },
+                    "IPv6": {
+                        "description": "Enable or Disable Pi-Hole IPv6 support. Enter: true or false",
+                        "label": "Enable IPv6",
+                        "index": 5,
+                        "default": "false"
+                    },
+                    "WEBPASSWORD": {
+                        "description": "Enter desired webpassword for pi-hole.",
+                        "label": "Web-Password",
+                        "index": 6
+                    }
+                }
+            }
+        },
+        "description": "PI-Hole by DigInc",
+        "more_info": "<h4>PI-HOLE™: A BLACK HOLE FOR INTERNET ADVERTISEMENTS</h4><p><b>Admin page</b></p><p>To access admin interface go to URL: http://[SERVERIP]/Admin</p><p>If you have different port than 80 you need to specify that in the URL.</p><p><b>Block Over 100,000 Ad-serving Domains</b></p><p>Known ad-serving domains are pulled from third party sources and compiled into one list.</p><p><b>Block Advertisements On Any Device</b></p><p>Network-level blocking allows any device to block ads, regardless of hardware or OS.</p><p><b>Improve Overall Network Performance</b></p><p>Since ads are blocked before they are downloaded, your network will perform better.</p>",
+        "ui": {
+            "slug": "admin"
```

Comment:
File should look like this afterward except the json extension not the txt extension.
[pi-hole.txt](https://github.com/rockstor/rockon-registry/files/1016514/pi-hole.txt)